src/pages/rooms/chat-input-form.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {
```

Comment:
Many functions in this file have no return type specified

Reply:
Done. Defined Return types of functions.